### PR TITLE
doc: intro doc updates

### PR DESCRIPTION
**Pull request by jmtaylor90**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

https://redmine.openinfosecfoundation.org/issues/5385

Describe changes:
- This work towards the goals stated in 5182.
-
-

#suricata-verify-pr:
#suricata-verify-repo:
#suricata-verify-branch:
#suricata-update-pr:
#suricata-update-repo:
#suricata-update-branch:
#libhtp-pr:
#libhtp-repo:
#libhtp-branch:

### CODE REVIEW

File: doc/userguide/rules/intro.rst [hunk 1]
```diff
@@ -100,21 +102,20 @@ you can pick from. These are:
 * sip
 * http2
 
-The availability of these protocols depends on whether the protocol is enabled in the configuration file suricata.yaml.
+The availability of these protocols depends on whether the protocol is enabled in the configuration file, suricata.yaml.
```
Comment (jufajardini) [inline doc/userguide/rules/intro.rst:105]: We are trying to enforce the <80 character limit for new changes, to comply with Sphinx recommendations. Can you make sure that the lines where you add changes are compliant?

 (this isn't a hard rule, I sometimes and up with 80 or 81)

File: doc/userguide/rules/intro.rst [hunk 2]
```diff
@@ -26,31 +26,28 @@ An example of a rule is as follows:
...
 In this example, :example-rule-action:`red` is the action,
 :example-rule-header:`green` is the header and :example-rule-options:`blue`
 are the options.
 
 We will be using the above signature as an example throughout
-this section, highlighting the different parts of the signature. It is a
-signature taken from the database of Emerging Threats, an open database
-featuring lots of rules that you can freely download and use in your
-Suricata instance.
+this section, highlighting the different parts of the signature.
```
Comment (jufajardini) [inline doc/userguide/rules/intro.rst:36]: Thoughts:

Even if we change the example rule, I think it's still useful/ valid to mention that ET has an open dataset that folks can use with Suricata (and/or learn from, when starting to write Suricata rules. Maybe even mention SV as a source of more rule examples).

File: doc/userguide/rules/intro.rst [hunk 3]
```diff
@@ -26,31 +26,28 @@ An example of a rule is as follows:
 
 .. container:: example-rule
 
-    :example-rule-action:`drop` :example-rule-header:`tcp $HOME_NET any -> $EXTERNAL_NET any` :example-rule-options:`(msg:"ET TROJAN Likely Bot Nick in IRC (USA +..)"; flow:established,to_server; flowbits:isset,is_proto_irc; content:"NICK "; pcre:"/NICK .*USA.*[0-9]{3,}/i"; reference:url,doc.emergingthreats.net/2008124; classtype:trojan-activity; sid:2008124; rev:2;)`
+    :example-rule-action:`alert` :example-rule-header:`http $HOME_NET any -> $EXTERNAL_NET any`  :example-rule-options:`(msg:"HTTP GET Request Containing Rule in URI"; flow:established,to_server; http.method; content:"GET"; http.uri; content:"rule"; fast_pattern; classtype:trojan-activity; sid:123; rev:1;)`
```
Comment (jufajardini) [inline doc/userguide/rules/intro.rst:29]: Shouldn't `classtype` be updated here, now?

**Review by jufajardini — CHANGES_REQUESTED** [review]
Overall it's looking good to me, thanks for your work!

I've left minor comments inline.

Only thing I'm not sure about is the example rule change. I've asked the team for input, and will try to give feedback on that asap :)

**Comment by jufajardini** [comment]
> Overall it's looking good to me, thanks for your work!
> 
> I've left minor comments inline.
> 
> Only thing I'm not sure about is the example rule change. I've asked the team for input, and will try to give feedback on that asap :)

Folks think the new rule has some advantages over the old one, so thanks for that, as well! =]

Could you leave somehow a comment about the ET open ruleset and SV tests as good places for more rule examples, for folks who are learning how to write their rules?

**Comment by jmtaylor90** [comment]
Thank you very much @jufajardini ! updated PR is https://github.com/OISF/suricata/pull/7548

**Pull request closed without merging**